microsoft-edge/webview2/release-notes.md

```diff
@@ -32,6 +32,106 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
+
+### General
+
+#### Experimental features
+
+* Added support to create a shared memory based buffer with the specified size:
+    *  `close`
+    *  `get_Buffer`
+    *  `get_FileMappingHandle`
+    *  `get_Size`
+    *  `OpenStream`  
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2SharedBuffer Class](/dotnet/api/microsoft.web.webview2.core.corewebview2sharedbuffer?view&preserve-view=true=webview2-dotnet-1.0.1466-prerelease)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2SharedBuffer Class](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalsharedbuffer?view&preserve-view=true=webview2-1.0.1466-prerelease)
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [CoreWebView2SharedBuffer Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2sharedbuffer?view&preserve-view=true=webview2-winrt-1.0.1466-prerelease
+)
+
+---
+
+*  Added support to share a shared buffer object with script of the main-frame/iframe
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2.PostSharedBufferToScript Method](/dotnet/api/microsoft.web.webview2.core.corewebview2.postsharedbuffertoscript?view&preserve-view=true=webview2-dotnet-1.0.1466-prerelease#microsoft-web-webview2-core-corewebview2-postsharedbuffertoscript(microsoft-web-webview2-core-corewebview2sharedbuffer-microsoft-web-webview2-core-corewebview2sharedbufferaccess-system-string))
+* [CoreWebView2Frame.PostSharedBufferToScript Method](/dotnet/api/microsoft.web.webview2.core.corewebview2frame.postsharedbuffertoscript?view&preserve-view=true=webview2-dotnet-1.0.1466-prerelease#microsoft-web-webview2-core-corewebview2frame-postsharedbuffertoscript(microsoft-web-webview2-core-corewebview2sharedbuffer-microsoft-web-webview2-core-corewebview2sharedbufferaccess-system-string))
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2.PostSharedBufferToScript method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2?view=&preserve-view=truewebview2-winrt-1.0.1466-prerelease#postsharedbuffertoscript)
+* [CoreWebView2Frame.PostSharedBufferToScript method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2frame?view&preserve-view=true=webview2-winrt-1.0.1466-prerelease#postsharedbuffertoscript)
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2Experimental18::PostSharedBufferToScript method](/microsoft-edge/webview2/reference/win32/icorewebview2experimental18?view=webview2-1.0.1466-prerelease&preserve-view=true#postsharedbuffertoscript)
+* [ICoreWebView2ExperimentalFrame4::PostSharedBufferToScript method](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalframe4?view=webview2-1.0.1466-prerelease&preserve-view=true#postsharedbuffertoscript)
+
+---
+
+*  Added support by adding these properties to run JavaScript code from the JavaScript parameter in the current top-level document:

```

Comment:
```suggestion
*  Added support for running JavaScript code from the `JavaScript` parameter in the current top-level document:
```
@peiche-jessica - is the name/spelling of the parameter:  `JavaScript`  ?

Reply:
I suppose from the context, since the API tab-set is immediately below, no need to say "by adding the following properties and/or methods:"

This is an aspect of doc design for cross-platform docs stylization.